mpi-mediator/importer/docker-compose.config.yml

```diff
@@ -4,9 +4,12 @@ services:
   # container for executing config import scripts for creating the OpenHIM channels used by the Mediator
   mpi-mediator-config-importer:
     image: node:erbium-alpine
+    networks:
+      openhim:
+      default:
     environment:
-      OPENHIM_CORE_SERVICE_NAME: ${OPENHIM_CORE_SERVICE_NAME}
-      OPENHIM_MEDIATOR_API_PORT: ${OPENHIM_MEDIATOR_API_PORT}
+      OPENHIM_CORE_SERVICE_NAME: 'openhim-core'
+      OPENHIM_MEDIATOR_API_PORT: 8080
```

Comment:
If you want to hardcode these I think you should remove the two env vars from package.json as well.
But just a question why did you hardcoded them ?

Reply:
Since the importers run from within the docker network, if you ever had to override these environment variables to lets say port 443 and example.jembi.org, the importer would then try and connect to https://example.jembi.org:443 which would not exist on the internal docker network yet (since the reverse proxy would only deploy later and would introduce a hard dependency on having the reverse proxy on openhim). 
So it doesn't make sense to override them to be the non-docker connection info, at least in this context. But agreed, I'll remove them and the other places that have the same convention.

Reply:
Actually we are hosting the core also, it is openhimcomms and we need to host it and we are using it as well, maybe you need to keep it as it is